### ISSUE TITLE
File Not Found Error

### ISSUE THREAD

**Issue by Dean-UQ** [Mathux/TEMOS#8]
Hi,
 Thank you for sharing the official codes. I followed the instructions in the README.md file. However, when I tried to run the train.py, I met a  _File Not Found Error_ and I would like to know what makes this error and how to fix it.

Many thanks for your kind help.

![微信截图_20220608201221](https://user-images.githubusercontent.com/97895595/172613728-29e46060-baba-406b-9e5e-c3cc770012d2.png)

**Comment by Mathux** [comment]
Hello,

It seems that your problem occur when the data module is being loaded: when it try to load few samples from the training set. 

To make sure your data is correctly prepared, can you send me the output of:
```
tree -d datasets
```
```
ls datasets/kit-splits/
```
and
```
find datasets/kit/* -name . -o -prune | head -20
```

**Comment by Dean-UQ** [comment]
Many thanks for your rapid reply.
Their outputs are:
![微信截图_20220608202448](https://user-images.githubusercontent.com/97895595/172631460-084d29e9-9ba4-4066-9044-e456a84f739b.png)

**Comment by Mathux** [comment]
I got the problem, you are missing the following files ``XXXXX_fke.csv``. 
I should make the README a bit more explicit. 

To be able to compare with prior works, we need to get the same exact same data. Both repository ([Ghosh](https://github.com/anindita127/Complextext2animation) and [JL2P](https://github.com/chahuja/language2pose)) are preprocessing the data to convert the axis-angle representation into xyz joints coordinates (fke means "forward kinematics") with a similar code (Ghosh is actually a fork of JL2P). 

Basically, you need to download their repository (one of them), and process the data with ``data.py``. Then you can copy the ``XXXXX_fke.csv`` files (or move the folder, or make a symlink).

**Comment by Dean-UQ** [comment]
Thanks for your kind help. I have preprocessed the raw data and copied the XXXXX_fke.csv files. But, I meet the  _File Not Found Error_ again.
![微信截图_20220609102355](https://user-images.githubusercontent.com/97895595/172751105-fa2b580b-75b1-477a-bf23-0cf3d0b9f3cc.png)

**Comment by Dean-UQ** [comment]
Many thanks for your kind help. This error is fixed.

**Comment by AkbarShah96** [comment]
Hi, I followed the data preprocessing steps in [JL2P](https://github.com/chahuja/language2pose) and then copied over the files to your repo. But I still get the same error as OP. 

The steps you mentioned above give the following results:
command: `tree -d datasets`
output: `datasets
├── AMASS
── kit
── kit-splits`

command: `ls datasets/kit-splits/`
output:`gtest  human-study  test  train  val  visu`

command: `find datasets/kit/* -name . -o -prune | head -20`
output: 
```datasets/kit/00001_annotations.json                                                                                  
datasets/kit/00001_meta.json                                                                                      
datasets/kit/00001_mmm.xml                                                                              
datasets/kit/00001_quat.csv                                                                                           
datasets/kit/00001_quat.fke                                                                                            
datasets/kit/00001_quat.rifke                                                                                          
datasets/kit/00001_raw.c3d                                                                                         
datasets/kit/00002_annotations.json                                                                                    
datasets/kit/00002_meta.json                                                                                          
datasets/kit/00002_mmm.xml                                                                                            
datasets/kit/00002_quat.csv                                                                                          
datasets/kit/00002_quat.fke                                                                                            
datasets/kit/00002_quat.rifke                                                                                          
datasets/kit/00002_raw.c3d                                                                                             
datasets/kit/00003_annotations.json                                                                                  
datasets/kit/00003_meta.json                                                                                          
datasets/kit/00003_mmm.xml                                                                                         
datasets/kit/00003_quat.csv                                                                                            
datasets/kit/00003_quat.fke                                                                                         
datasets/kit/00003_quat.rifke
